src/App.css

```diff
@@ -185,7 +180,28 @@ body {
     padding: 0 1em;
     width: 100%;
 }
+h1{ color:#285E61; text-align: center; font-family: Verdana; 
```

Comment:
Is this style for working with the chakra <Header> component or  h1 tags somewhere in the code?

Reply:
that belongs to the "Open Games List"  tittle in OpenGamesPage.js  page